views/forms/ChartFilterForm/ChartFilterForm.css

```diff
@@ -21,3 +21,13 @@
   width: 158px;
   padding-left: 15px;
 }
+
+@media screen and (max-width: 1040px) {
```

Comment:
I think we should keep our breakpoints as variables to make sure they're consistent, but that may be a separate refactor ticket so up to you!